diff --git a/udagram-frontend/src/environments/environment.prod.ts b/udagram-frontend/src/environments/environment.prod.ts
index a19a0e1..183b725 100755
--- a/udagram-frontend/src/environments/environment.prod.ts
+++ b/udagram-frontend/src/environments/environment.prod.ts
@@ -5,8 +5,7 @@
 export const environment = {
   production: false,
   appName: 'Udagram',
-  //apiHost: 'http://localhost:8080/api/v0'
-  apiHost: 'http://adae377b9f246447a8b089f3d1e7cadf-339208073.us-west-1.elb.amazonaws.com:8080/api/v0'
+  apiHost: 'http://localhost:8080/api/v0'
 };
 
 /*
diff --git a/udagram-frontend/src/environments/environment.ts b/udagram-frontend/src/environments/environment.ts
index 183b725..54c4e02 100755
--- a/udagram-frontend/src/environments/environment.ts
+++ b/udagram-frontend/src/environments/environment.ts
@@ -5,7 +5,7 @@
 export const environment = {
   production: false,
   appName: 'Udagram',
-  apiHost: 'http://localhost:8080/api/v0'
+  apiHost: 'http://a1c0a229a0086441aaa3277d67918a9a-1373844705.us-west-1.elb.amazonaws.com:8080/api/v0'
 };
 
 /*
